collections/guides/manage_applications_terraform/index.md

```diff
@@ -0,0 +1,154 @@
+  api_key        = "<YOUR_API_KEY_HERE>"
+  client_timeout = 5
+}
+```
+
+Another configuration option is to set the API URL and key through setting `BASISTHEORY_API_URL` and `BASISTHEORY_API_KEY`
+environment vars. This leaves you with optionally setting the client timeout, which defaults to 15 seconds. So the most
+minimal `main.tf` config can look like this:
+
+```terraform
+provider "basistheory" {
+}
+```
+
+Additionally, you can pin down the version of the Terraform provider you use for the configuration with the following `terraform`
+block in the same file:
+
+```terraform
+terraform {
+  required_providers {
+    basistheory = {
+      source = "basistheory/basistheory"
+      version = "1.0.0"
+    }
+  }
+}
+```
+
+### Set up the basistheory_application resource
+
+Typically, resources are set up from the `provider` and `terraform` block. So let's create an `application.tf` file with
+the configuration for our Application. Feel free to modify the following configuration for your use case:
+
+```terraform
+resource "basistheory_application" "my_application" {
+  name = "My Application"
+  type = "server_to_server"
+  permissions = [
+    "token:general:create",
+    "token:general:read:low",
+    "token:pci:create",
+    "token:pci:read:low",
+  ]
+}
+```
+
+### Set up the output for your Application key
+
+To make use of the Application you create, you'll need to extract the key. Defining `output` blocks make data about your
+resources available for use within the command line. Define the `ouput` block in a new `output.tf` file with the contents
+below:
+
+```terraform
+output "my_application_key" {
+  value       = basistheory_application.my_application.key
+  description = "My Application key"
+  sensitive   = true
+}
+```
+
+## Executing Terraform
+
+Now that our configuration is set up, it's time to execute Terraform commands to get these resources created.
+
+The first step in managing any resources via Terraform, is to initialize your directory with your configuration. There are
+many things that occur when initializing Terraform, one of which is download the `basistheory` provider from the public
+Terraform registry. Let's run the following `terraform` commands within the same directory containing your configuration.
+
+```bash
+terraform init
+```
+
+A prudent step to ensure you have a valid configuration is to run `basistheory`'s provider validations against your current
+configuration. These configurations only uses local resources to validate against the downloaded provider and does not talk
+to any remote resources. 
+
+```bash
+terraform validate
+```
+
+Finally, you can apply the configuration using the `apply` command. This will output a plan Terraform intends to execute
+after confirming the plan. The plan contains differences between your current configuration and your current Terraform state.
+After executing the following command, type `yes` to create your Application within Basis Theory.
```

Comment:
```suggestion
Finally, you can apply the configuration using the `apply` command. `apply` will output a plan Terraform intends to execute
and waits for your confirmation. This plan contains differences between your current configuration and your current Terraform state.
After reviewing the plan, type `yes` to create your Application within Basis Theory.
```

Reply:
this may still read a bit odd. could use some eyes still

Reply:
I think your suggestions make sense